modules/transactions.js

```diff
@@ -187,6 +187,33 @@ __private.getById = function (id, cb) {
 	});
 };
 
+__private.getVotesById = function (transaction, cb) {
+	library.db.query(sql.getVotesById, {id: transaction.id}).then(function (rows) {
+		if (!rows.length) {
+			return setImmediate(cb, 'Transaction not found: ' + id);
+		}
+
+		var votes = rows[0].votes.split(',');
+		var added = [];
+		var deleted = [];
+
+		for (var i = 0; i < votes.length; i++) {
+			if (votes[i].substring(0, 1) == "+") {
```

Comment:
Please consider string indexing ([here](https://jsperf.com/index-vs-slice-vs-substring) is a performance test).